### PR TITLE
Updated Max pool1d

### PR DESCRIPTION
**Pull request by mtnvdsk**
Adding max_pool1d #15775

### CODE REVIEW

**Comment by mtnvdsk** [comment]
close #15776

**Comment by mtnvdsk** [comment]
sir @kurshakuz , All the test cases are passing except a jax test case failure could you please guide me to rectify the error and code.

File: ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py [hunk 1]
```diff
@@ -1541,32 +1541,25 @@ def arrays_for_pooling(
...
+        else:
+            kernel.append(draw(st.integers(1, 10)))  # Replace 10 with an appropriate maximum value
+
+        if return_dilation:
             if kernel[i] > 1:
-                max_dilation = (in_shape[i + 1] - kernel[i]) // (kernel[i] - 1) + 1
+                max_dilation = (in_shape[i+1] - kernel[i]) // (kernel[i] - 1) + 1
                 dilations.append(draw(st.integers(1, max_dilation)))
-                new_kernel.append(kernel[i] + (kernel[i] - 1) * (dilations[i] - 1))
+                new_kernel.append(kernel[i] + (kernel[i] - 1) * (dilations[i-1] - 1))
```
Comment (CatB1t) [inline ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py:1558]: Hello @mtnvdsk , can you please explain the rationale behind those changes? what is the bug/issue you're trying to fix?

**Review by CatB1t — CHANGES_REQUESTED** [review]

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**